packages/core/src/modules/credentials/formats/jsonld/JsonLdCredentialFormat.ts

```diff
@@ -11,15 +11,29 @@ export interface SignCredentialOptionsRFC0593 {
   options: JsonLdOptionsRFC0593
 }
 
+export interface SignCredentialOptionsRFC0593AsJson {
```

Comment:
xCan we remove the AsJson part and include JsonLd in the name?

```suggestion
export interface JsonLdSignCredentialFormat {
```